### ISSUE TITLE
More Colors button is not read properly by screen readers

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor4#5218]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open https://ckeditor.com/docs/ckeditor4/latest/examples/fullpreset.html sample with screen reader enabled 
2. Put focus on the toolbar (Alt+F10) and navigate to Color or Background Button
3. Open it and move to More Colors

### Expected result

Screen reader should announce it as button (perhaps a button which opens dialog)

### Actual result

Screen readers announce it as "More Colors 26/26"

## Other details

I have checked NVDA, Narrator, Voice Over and JAWS.
Support issue no. 74678

* Browser: Any
* OS: Any
* CKEditor version: 4.0+
* Installed CKEditor plugins: Color Button

**Comment by jacekbogdanski** [comment]
By @Comandeer:

> The menu that is opened after pressing one of the buttons for selecting colors is marked up with [the listbox role](https://w3c.github.io/aria/#listbox) – as they represent the list of options that the user can choose from. The only role for children of the listbox widget is [the option one](https://w3c.github.io/aria/#option) which is used for all elements inside the menu. The markup of the color selection menu seems to be compliant with the accessibility standards.
 
> Additionally, marking up the "More colors…" button with a different role (e.g. button) could cause incorrect and inconsistent behavior between screen readers, e.g. some screen readers would no longer announce that the button is the nth element in the n-element collection ("More colors…, 26 of 26" → "More colors…, button") but still announce it for every other element inside the color selection menu.

Therefore, I'm closing the issue as "by design" for now.
